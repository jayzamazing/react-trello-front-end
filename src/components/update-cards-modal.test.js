'use strict';
import React from 'react';
import toJson from 'enzyme-to-json';
import {shallow} from 'enzyme';
import UpdateCardsModal from './update-cards-modal';
//testing for the create-board-modal
describe('update-cards-modal', () => {
  it('should render', () => {
    shallow(<UpdateCardsModal isOpen={true}/>);
  });
  //take snapshot
  it('snapshot should match', () => {
    const wrapper = shallow(<UpdateCardsModal isOpen={true}/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
