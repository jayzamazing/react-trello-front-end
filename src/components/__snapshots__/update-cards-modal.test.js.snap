// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`update-cards-modal snapshot should match 1`] = `
<Modal
  ariaHideApp={true}
  bodyOpenClassName="ReactModal__Body--open"
  className={
    Object {
      "afterOpen": "update-cards-modal-after-open",
      "base": "update-cards-modal",
      "beforeClose": "update-cards-modal-before-close",
    }
  }
  closeTimeoutMS={0}
  contentLabel="Update Cards Modal"
  isOpen={true}
  onRequestClose={[Function]}
  overlayClassName={
    Object {
      "afterOpen": "update-cards-modal-overlay-after-open",
      "base": "update-cards-modal-overlay",
      "beforeClose": "update-cards-modal-overlay-before-close",
    }
  }
  parentSelector={[Function]}
  portalClassName="ReactModalPortal"
  shouldCloseOnEsc={true}
  shouldCloseOnOverlayClick={true}
  shouldFocusAfterRender={true}
  shouldReturnFocusAfterClose={true}
>
  <button
    aria-label="close button"
    className="btn btn-default close-modal-btn"
    onClick={[Function]}
    type="button"
  >
    <span
      aria-hidden="true"
      className="glyphicon glyphicon-remove"
    />
  </button>
  <Connect(ReduxForm)
    form="undefined-2"
  />
  <span
    className="glyphicon glyphicon-minus cards-delete"
    onClick={[Function]}
  />
  <Connect(ReduxForm)
    form="undefined-3"
  />
  <span
    className="glyphicon glyphicon-minus cards-update"
    onClick={[Function]}
  />
</Modal>
`;
